packages/wallet/api/src/findOrMakeInvitation.js

```diff
@@ -94,9 +94,23 @@ const makeContinuingInvitation = async (
   return invitationP;
 };
 
+const makeInvitation = async (
+  invitationMaker,
+  instanceHandleBoardId,
+  board,
+  zoe,
+) => {
+  const instance = E(board).getValue(instanceHandleBoardId);
+  const publicFacet = E(zoe).getPublicFacet(instance);
+  const { description, args = [] } = invitationMaker;
```

Comment:
so `description` specifies a method name? this seems like an abuse of the field. why is this safe?

Reply:
I'm not sure what you mean by safe, is `description` misleading of a name? Maybe `descriptor` or `method` instead?

The user has to approve this, so I don't see it as being less safe than the dapp passing in an arbitrary invitation like the other way.

Reply:
yeah, `method` would address my concern. `description` sounds like a general field for arbitrary text but the string in this field has powers, specifically to specify the method to call.